lib/src/constants.dart

```diff
@@ -8,6 +8,10 @@ import 'state/grid.dart';
 // WARNING: Do not modify line below, except for the version string.
 const String CURRENT_VERSION = "0.13.0";
 const String INITIAL_VERSION = "0.1.0";
+final scadnano_versions = [
```

Comment:
Let's rename this variable to `scadnano_versions_to_link`, since it's not actually all the scadnano versions.

Reply:
Resolved in https://github.com/UC-Davis-molecular-computing/scadnano/pull/510/commits/d19ac32b88656de64e26a42c8e96e27cd0ef15ee